app/assets/stylesheets/layouts/_dark_selectors.scss

```diff
@@ -87,21 +125,26 @@ table { border-color: #555; } // TODO: what does this style
 .icon-credit { background-color: $bg_color_icon_credit; }
 
 /* - tag items */
-.tag-item, tr.gallery-tags .tag-item { background-color: $bg_color_tag_item; color: $font_color_tag_item; }
+.tag-item, .gallery-tags .tag-item {
+  background-color: $bg_color_tag_item;
+  color: $font_color_tag_item;
+}
 
 /* - link-boxes */
-.link-box.action-new { color: $font_color_linkbox_new; }
-.link-box.action-edit { color: $font_color_linkbox_edit; }
-.link-box.action-favorite { color: $font_color_linkbox_favorite; }
-.link-box.action-delete { color: $font_color_linkbox_delete; }
-.link-box.action-dismiss { color: $font_color_linkbox_dismiss; }
+.link-box {
+  .action-new { color: $font_color_linkbox_new; }
+  .action-edit { color: $font_color_linkbox_edit; }
+  .action-favorite { color: $font_color_linkbox_favorite; }
+  .action-delete { color: $font_color_linkbox_delete; }
+  .action-dismiss { color: $font_color_linkbox_dismiss; }
+}
```

Comment:
Think you want `&` on each of the rules inside here, since currently they're turning into:

`.link-box .action-new` instead of `.link-box.action-new`
`.link-box .action-edit` instead of `.link-box.action-edit`
etc

i.e. you want:

```scss
.link-box {
  &.action-new { color: $font_color_linkbox_new; }
  &.action-edit { color: $font_color_linkbox_edit; }
  &.action-favorite { color: $font_color_linkbox_favorite; }
  &.action-delete { color: $font_color_linkbox_delete; }
  &.action-dismiss { color: $font_color_linkbox_dismiss; }
}
```

Reply:
Oh. You did that in the next commit.

Reply:
Mind squashing this & relevant parts of the next commit?